cspell.config.yaml

```diff
@@ -11,8 +11,7 @@ dictionaries:
   - "!backwards-compatibility"
   - "!cryptocurrencies"
   - "!cpp"
-flagWords:
-  - ADT
+flagWords: []
```

Comment:
Probably best to leave this in and remove it from the code.

Related: https://github.com/ansible/ansible-dev-tools/issues/149